VideoLocker/src/main/java/org/edx/mobile/view/CourseDiscussionPostsThreadFragment.java

```diff
@@ -234,32 +231,6 @@ public void onDestroy() {
         EventBus.getDefault().unregister(this);
     }
 
-    @Override
-    public void onRestart() {
-        if (postsSort == DiscussionPostsSort.LAST_ACTIVITY_AT && mSelectedItem != -1) {
-            // Move the last viewed thread to the top
-            discussionPostsAdapter.moveToTop(mSelectedItem);
```

Comment:
I've kept the `moveToTop` function on purpose, as it might come in handy in the future.